pkg/controllers/job/plugins/factory.go

```diff
@@ -20,7 +20,7 @@ import (
 	"sync"
 
 	"volcano.sh/volcano/pkg/controllers/job/plugins/env"
-	"volcano.sh/volcano/pkg/controllers/job/plugins/interface"
+	pluginsinterface "volcano.sh/volcano/pkg/controllers/job/plugins/interface"
```

Comment:
Modifying `pkg/controllers/job/plugins/factory.go#L40` might be a better choice.

Reply:
no, this is `goimports` specification, 
the package name is `pluginsinterface` and the last part of import path `volcano.sh/volcano/pkg/controllers/job/plugins/interface` is `interface`, the two are not equal. 
On this senario, we should use alias. 
If not, goimports will failed with information: 
File is not `goimports`-ed with -local volcano.sh (goimports)